app/views/reports/_auditing_report.html.haml

```diff
@@ -46,19 +46,25 @@
               =label_tag "title", t(:documents)[:title]+t(:actions)[:required], class: "col-sm-3 control-label"
               .col-sm-9= text_field_tag "title", title, class: "form-control"
             .form-group
-              = label_tag "start_date", t(:reports)[:start_date]+t(:actions)[:required], class: "col-sm-3 control-label"
-              .col-sm-9
-                = text_field_tag "start_date", nil, class: "datepicker form-control", readonly: true
+              = label_tag "start_date", t(:reports)[:start_date]+t(:actions)[:required], class: "col-sm-4 col-form-label"
+              .input-group.datetimepicker.date.col-sm-8{ id: 'start_date', data: {target_input: 'nearest' }}
+                = text_field_tag "start_date", nil, class: "form-control datetimepicker", readonly: true, data: { target:"#start_date" }
+                .input-group-append{ data: { toggle: 'datetimepicker', target: '#start_date' }}
+                  .input-group-text
+                    %span.far.fa-calendar
             .form-group
-              = label_tag "end_date", t(:reports)[:end_date]+t(:actions)[:required], class: "col-sm-3 control-label"
-              .col-sm-9
-                = text_field_tag "end_date", nil, class: "datepicker form-control", readonly: true
+              = label_tag "end_date", t(:reports)[:end_date]+t(:actions)[:required], class: "col-sm-4 col-form-label"
```

Comment:
Also for the * indicator for required fields, can you add the `required` class on the label instead of appending the text. That will add a stylized red asterisk and is also part of inline form validation messages

Reply:
Done.